samples/kvsWebRTCClientMasterGstreamerSample.c

```diff
@@ -140,6 +140,15 @@ PVOID sendGstreamerAudioVideo(PVOID args)
      * videotestsrc is-live=TRUE ! video/x-raw,width=1280,height=720,framerate=30/1 !
      * vp8enc error-resilient=partitions keyframe-max-dist=10 auto-alt-ref=true cpu-used=5 deadline=1 !
      * appsink sync=TRUE emit-signals=TRUE name=appsink-video
+     *
+     * Raspberry Pi Hardware Encode Example
+     * "v4l2src do-timestamp=TRUE device=\"/dev/video0\" ! queue ! videoconvert ! "
+     * "video/x-raw,format=I420,width=640,height=480,framerate=30/1 ! "
+     * "omxh264enc control-rate=1 target-bitrate=500000 periodicty-idr=30 inline-header=FALSE ! "
+     * "h264parse config-interval=-1 ! "
```

Comment:
Is it necessary to send sps/pps with every key frame?

Reply:
Honestly, I'm not sure.  I know most camera vendors send sps/pps withe very keyframe, but that is for rtsp/rtp.